src/unity/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -790,7 +790,7 @@ def export_coreml(self, path, image_shape=(256, 256)):
 
         # append batch size and channels
         image_shape = (1, 3) + image_shape
-        c_image = _mx.sym.Variable('image', shape=image_shape,
+        c_image = _mx.sym.Variable(self.content_feature, shape=image_shape,
```

Comment:
Can we add a simple unit test without "image" hard coded or change the current unit test to not have "image" just to be sure.